src/Sylius/Behat/Page/Admin/Product/IndexPerTaxonPage.php

```diff
@@ -21,7 +21,11 @@ class IndexPerTaxonPage extends CrudIndexPage implements IndexPerTaxonPageInterf
 {
     public function getProductPosition(string $productName): int
     {
-        return (int) $this->getElement('product_taxon_position', ['%name%' => $productName])->getValue();
+        /** @var NodeElement $productsRow */
+        $productsRow = $this->getElement('table')->find('css', sprintf('tbody > tr:contains("%s")', $productName));
+        Assert::notNull($productsRow, 'There are no row with given product\'s name!');
+
+        return (int) $productsRow->find('css', '.sylius-product-taxon-position')->getValue();
```

Comment:
It would be good not to rely on the CSS class

Reply:
Usually, I'd be for it, but this class is a functional hook for the stimulus controller so in this case it's either or.